components/BlurryPoster/BlurryPoster.tsx

```diff
@@ -123,9 +124,9 @@ export const BlurryPoster = ({
           alt={bgImageAlt || ''}
           width={2000}
           height={1200}
-          aria-describedby={hasCaption ? 'story-hero-caption' : undefined}
+          aria-describedby={hasCaption && !!bgImageAlt ? 'story-hero-caption' : undefined}
           className={styles.bgImage}
-          fetchPriority="high"
+          fetchPriority={type === 'hero' ? 'high' : 'auto'}
```

Comment:
Only add aria-describedby to the hero background image if it has an alt.
Don't use fetchPriority high unless this component is used for the Story Hero, ie, when used on the homepage as featured blurry posters, don't use high priority